truss-chains/truss_chains/utils.py

```diff
@@ -177,6 +177,22 @@ def populate_chainlet_service_predict_urls(
     return chainlet_to_deployed_service
 
 
+# NOTE: This needs to be available in the Context Builder
+# so that older Truss CLI versions that generate code that
+# expects this function to be available continue to work.
```

Comment:
So the real condition here is the chains version that's added to the chainlet's pip-requirements, right? You could still deploy chains from older CLIs if you manually added a newer version, right?

Reply:
Yeah that should be correct.